### PR TITLE
Regression Test Failures for 121

### PR DESCRIPTION
**Pull request by bwelch8105**
Fixed some of the regression test failures and removed the test file around the Review Info tab that has, now, been removed.

### CODE REVIEW

File: cypress/Shared/MeasureGroupPage.ts [hunk 1]
```diff
@@ -130,10 +129,10 @@ export class MeasureGroupPage {
     public static readonly denominatorObservation = '[id="measure-observation-denominator"]'
     public static readonly denominatorAggregateFunction = '[id="measure-observation-aggregate-denominator"]'
     public static readonly removeDenominatorObservation = '[data-testid="measure-observation-denominator-remove"]'
-    public static readonly measureObservationSelect = '[class="MuiMenuItem-root MuiMenuItem-gutters MuiButtonBase-root css-1km1ehz"]'
+    public static readonly measureObservationSelect = '[class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiPaper-root MuiMenu-paper MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiPopover-paper css-177ic5c"]'
```
Comment (cvasile001) [inline cypress/Shared/MeasureGroupPage.ts:132]: is there a better name for this?

File: cypress/Shared/MeasureGroupPage.ts [hunk 2]
```diff
@@ -130,10 +129,10 @@ export class MeasureGroupPage {
...
     public static readonly aggregateFunctionCount = '[data-value="Count"]'
     public static readonly aggregateFunctionMaximum = '[data-value="Maximum"]'
-    public static readonly aggregateFunctionDropdownList = '[class="MuiMenuItem-root MuiMenuItem-gutters MuiButtonBase-root css-1km1ehz"]'
+    public static readonly aggregateFunctionDropdownList = '[class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiPaper-root MuiMenu-paper MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiPopover-paper css-177ic5c"]'
```
Comment (cvasile001) [inline cypress/Shared/MeasureGroupPage.ts:135]: same here

File: cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts [hunk 3]
```diff
@@ -960,41 +960,11 @@ describe('Supplemental data elements and Risk Adjustment variables on Measure gr
...
-        cy.get(MeasureGroupPage.RAPage)
-            .find('[class="right"]')
-            .find(MeasureGroupPage.RAPageContainer)
-            .find(MeasureGroupPage.riskAdjustmentDescriptionTextBox)
-            .last() // select the last element
-            .prev()
-            .should('contain.text', 'Denominator Description')
     })
     //skipping until bug MAT-5829 is fixed
     it.skip('Verify that description entered for each SDE Definition is accurate', () => {
```
Comment (cvasile001) [inline cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts:970]: I think we can unskip this now

**Review by cvasile001 — CHANGES_REQUESTED** [review]
left a few comments there

File: cypress/Shared/MeasureGroupPage.ts (hunk 1 above)
Comment (bwelch8105) [reply cypress/Shared/MeasureGroupPage.ts:132]: Changed how I was accessing the observation values

File: cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts (hunk 3 above)
Comment (bwelch8105) [reply cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts:970]: un-skipped this test and it ran fine in local

**Review by cvasile001 — CHANGES_REQUESTED** [review]

**Review by cvasile001 — APPROVED** [review]
approved

**Pull request merged**